aspen/http/resource.py

```diff
@@ -57,39 +57,41 @@ def render(self, state):
         we can ignore it: <https://tools.ietf.org/html/rfc7231#section-5.3.2>).
         """
         available = self.available_types
+        choice = 0  # default to first available type
```

Comment:
You're going in circles with this, `choice` should be a media type, not an index of the list of media types.